nibabel/streamlines/tests/test_tractogram.py

```diff
@@ -181,6 +181,21 @@ def test_getitem(self):
             assert_arrays_equal(sdict[-1][k], v[-1])
             assert_arrays_equal(sdict[[0, -1]][k], v[[0, -1]])
 
+    def test_extend(self):
```

Comment:
By choice, `PerArrayDict` and `PerArraySequenceDict` don't support `+` and `+=`. Only `Tractogram` objects have it.